src/flow.jl

```diff
@@ -199,6 +206,12 @@ function update(sf::SurfaceFlowLand, network, frac_toriver)
     @. sf.alpha = sf.alpha_term * pow(sf.width, sf.alpha_pow)
     @. sf.qlat = sf.inwater / sf.dl
 
+    # Convert threshold to volume
+    threshold_volume = sf.h_thresh .* sf.width .* sf.dl
+
+    # Get pond volume at the start of the time step
+    pond_volume_current = sf.pond_height .* sf.width .* sf.dl
```

Comment:
This will result in more allocation and could have an impact on performance. Would be nice if this can be avoided (in the for loop?).